packages/core/helper-plugin/lib/src/components/GenericInput/index.js

```diff
@@ -134,6 +136,23 @@ const GenericInput = ({
     : '';
 
   switch (type) {
+    case 'json': {
+      return (
+        <JSONInput
+          label={label}
+          labelAction={labelAction}
+          value={value}
+          error={errorMessage}
+          hint={hint}
+          required={required}
+          onChange={(json) => {
+            onChange({ target: { name, value: json } });
+          }}
+          minHeight={pxToRem(253)}
+          maxHeight={pxToRem(506)}
```

Comment:
@maevalienard Should we use that moment to align the JSON Input to our 4point grid? :) Would you have input on how we could improve these values?

Reply:
We can make it:

minHeight={pxToRem(252)}
maxHeight={pxToRem(504)}